modules/gdscript/gdscript_parser.h

```diff
@@ -50,6 +50,31 @@
 #include "gdscript_warning.h"
 #endif // DEBUG_ENABLED
 
+static Variant::Type _variant_type_to_typed_array_element_type(Variant::Type p_type) {
+	switch (p_type) {
+		case Variant::PACKED_BYTE_ARRAY:
+			[[fallthrough]];
+		case Variant::PACKED_INT32_ARRAY:
+			[[fallthrough]];
+		case Variant::PACKED_INT64_ARRAY:
+			return Variant::INT;
```

Comment:
You don't really need `[[fallthrough]]` here since there are no other statements. It's easier to read if the cases are packed together.`

```suggestion
	switch (p_type) {
		case Variant::PACKED_BYTE_ARRAY:
		case Variant::PACKED_INT32_ARRAY:
		case Variant::PACKED_INT64_ARRAY:
			return Variant::INT;
```

Reply:
I agree with yout point, but some of the CI build checks would fail otherwise.

Reply:
I don't see why the CI would fail, we have already many instances of this in the code.

Reply:
I don't remember how I managed to come to believe that those were needed... In any case, I've removed them and all seems fine!